processor/transformprocessor/factory_test.go

```diff
@@ -33,15 +42,82 @@ func TestFactory_CreateDefaultConfig(t *testing.T) {
 	assert.Equal(t, cfg, &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewComponentID(typeStr)),
 		Traces: TracesConfig{
-			Queries: []string{},
+			Queries: []traces.Query{},
 		},
 	})
 	assert.NoError(t, configtest.CheckConfigStruct(cfg))
 }
 
+func TestFactory_LoadConfig(t *testing.T) {
```

Comment:
The way how this is written, is more or less a test for the processor now. May consider to add a `Equal` method on the `Query` to simplify load config test?